### ISSUE TITLE
User needs right to execute Voipmanager to execute Phone app

### ISSUE THREAD

**Issue by j-a-n** [tine20/tine20#6909]
Since version 2018.08.3 a user can't use the Phone app without the right to execute Voipmanager.
A user with right to execute Voipmanager is able to add and delete phones.

**Comment by pschuele** [comment]
yep, thanks for opening the issue. this is a result of the latest javascript loader improvements (see https://github.com/tine20/tine20/commit/08edd33c4498e724349f98c56eb27104482db0eb) and will be fixed soon.
